src/db.rs

```diff
@@ -50,6 +50,12 @@ unsafe impl Send for DB {}
 // use within the rocksdb library is generally behind a const reference
 unsafe impl Sync for DB {}
 
+// Option that specifies whether open DB for read only.
+enum RWOption {
```

Comment:
Does it make sense to add derives Clone and Copy here ?

Reply:
And I'd rename it to `AccessType`.

Reply:
> Does it make sense to add derives Clone and Copy here ?

Probably not necessary right now. It's not a public type so we can always add later if needed. I'll rename it.